src/language/languageUnderstanding.js

```diff
@@ -0,0 +1,52 @@
+const commonService = require('../commonService');
+
+/**
+ * Language Understanding API is a cloud-based service that provides advanced natural language processing over raw text, and intent and entity detection.
+ * Your LUIS domain-specific model must be in built, trained, and published before using this endpoint.
+ * @augments commonService
+ * {@link https://docs.microsoft.com/en-us/azure/cognitive-services/luis/home}
+ */
+class languageUnderstanding extends commonService {
+    /**
+     * Constructor.
+     * 
+     * @param {Object} obj
+     * @param {string} obj.apiKey
+     * @param {string} obj.endpoint
+     */
+    constructor({ apiKey, appID, endpoint }) {
+        super({ apiKey, endpoint });
+        this.appID = appID;
+        this.serviceId = "LUIS.v2.0"
+        this.endpoints = [
+            "westus.api.cognitive.microsoft.com",
+            "eastus2.api.cognitive.microsoft.com",
+            "westcentralus.api.cognitive.microsoft.com",
+            "westeurope.api.cognitive.microsoft.com",
+            "southeastasia.api.cognitive.microsoft.com"
+        ];
+    }
+
+    /**
+     * Returns the detected intent, entities and entity values with a score for each intent. 
+     * Scores close to 1 indicate 100% certainty that the identified intent is correct. 
+     * Irrespective of the value, the intent with the highest score is returned.
+     * @returns {Promise.<object>}
+     */
+    detectIntent({ headers, body }) {
+
+        const operation = {
+            "path": "luis/v2.0/apps/" + this.appID,
+            "method": "POST"
```

Comment:
This method is [this endpoint](https://westus.dev.cognitive.microsoft.com/docs/services/5819c76f40a6350ce09de1ac/operations/5819c77140a63516d81aee79), right? There are query string parameters that you didn't include and the `content-type` header. Something like this: https://github.com/joshbalfour/node-cognitive-services/blob/master/src/vision/face.js#L56